client/commands/v2/pysa_server.py

```diff
@@ -0,0 +1,546 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+"""
+This is a temporary implementation of Pysa's langauge server. This is an exact copy
+of Pyre's langauge server, which lives in persistent.py. 
+"""
+
+import asyncio
+import dataclasses
```

Comment:
Tons of unused imports here:

- dataclass
- enum
- json
- subprocess
- tempfile
- Set
- Union
- List
- version x2
- statistics
- server_event

You can set up `flake8` to catch these